### PR TITLE
bi fpn without detach

### PR DESCRIPTION
**Pull request by rydenisbak**
I got better result on face detection task with this changes.
Overall detach seems strange for me. We add inputs like skip connections and why we need to detach them?

### CODE REVIEW

**Comment by toandaominh1997** [comment]
@rydenisbak Thanks for your pull request, I have read your code, I feel it makes no difference. Can you explain this difference?

78978279_578935422907320_8238087675656011776_n

**Comment by rydenisbak** [comment]
Hello, oh excuse me. I cloned your repo two days ago and saw version with this code
```python
for in_tensor in inputs:
    pathtd.append(in_tensor.clone().detach())
```
it was seemed like mistake, but then you fixed it.

Now, we have no difference
